### PR TITLE
feat(crowdin): add arabic language to download workflow

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Comment by Sboonny** [comment]
welp. 

Can't formatting it locally with prettier doesn't change the file 🤷

File: .github/workflows/crowdin-download.client-ui.yml [hunk 1]
```diff
@@ -238,6 +238,35 @@ jobs:
...
+      
+          push_translations: false
+      
+          # pull-request
+          create_pull_request: false
+      
+          # global options
+          config: './crowdin-config.yml'
+          base_url: ${{ secrets.CROWDIN_BASE_URL_FCC }}
+
```
Comment (naomi-lgbt) [inline .github/workflows/crowdin-download.client-ui.yml:266]:
```suggestion

```

File: .github/workflows/crowdin-download.client-ui.yml [hunk 2]
```diff
@@ -238,6 +238,35 @@ jobs:
...
+          import_eq_suggestions: false
+      
+          # downloads
+          download_translations: true
+          download_language: arabic
+          skip_untranslated_files: false
+          export_only_approved: true
+      
+          push_translations: false
+
```
Comment (naomi-lgbt) [inline .github/workflows/crowdin-download.client-ui.yml:259]:
```suggestion

```

File: .github/workflows/crowdin-download.client-ui.yml [hunk 3]
```diff
@@ -238,6 +238,35 @@ jobs:
...
+          upload_translations: false
+          auto_approve_imported: false
+          import_eq_suggestions: false
+      
+          # downloads
+          download_translations: true
+          download_language: arabic
+          skip_untranslated_files: false
+          export_only_approved: true
+
```
Comment (naomi-lgbt) [inline .github/workflows/crowdin-download.client-ui.yml:257]:
```suggestion

```

File: .github/workflows/crowdin-download.client-ui.yml [hunk 4]
```diff
@@ -238,6 +238,35 @@ jobs:
...
+      - name: Crowdin Download Arabic Translations
+        uses: crowdin/github-action@master
+        # options: https://github.com/crowdin/github-action/blob/master/action.yml
+        with:
+          # uploads
+          upload_sources: false
+          upload_translations: false
+          auto_approve_imported: false
+          import_eq_suggestions: false
+
```
Comment (naomi-lgbt) [inline .github/workflows/crowdin-download.client-ui.yml:251]:
```suggestion

```

File: .github/workflows/crowdin-download.client-ui.yml [hunk 5]
```diff
@@ -238,6 +238,35 @@ jobs:
...
+          download_translations: true
+          download_language: arabic
+          skip_untranslated_files: false
+          export_only_approved: true
+      
+          push_translations: false
+      
+          # pull-request
+          create_pull_request: false
+
```
Comment (naomi-lgbt) [inline .github/workflows/crowdin-download.client-ui.yml:262]:
```suggestion

```

File: .github/workflows/crowdin-download.client-ui.yml [hunk 6]
```diff
@@ -238,6 +238,35 @@ jobs:
...
+        with:
+          # uploads
+          upload_sources: false
+          upload_translations: false
+          auto_approve_imported: false
+          import_eq_suggestions: false
+      
+          # downloads
+          download_translations: true
+          download_language: arabic
```
Comment (naomi-lgbt) [inline .github/workflows/crowdin-download.client-ui.yml:254]:
```suggestion
          download_language: ar
```

This needs to be the two character ISO code.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Trailing spaces seem to be the lint issue.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Okay this config looks good now - let's mirror these changes in the `crowdin-download.curriculum.yml` as well.

**Review by naomi-lgbt — DISMISSED** [review]
Okay this should be good to go.

**Review by naomi-lgbt — APPROVED** [review]

**Review by raisedadead — APPROVED** [review]

**Pull request merged**